client/src/components/questions.tsx

```diff
@@ -64,8 +64,14 @@ function Questions(props: {
     return orderedQuestions;
   });
 
+  const videoChatProps: ChatProps = {
+    displayMentorNames: true,
+    height: 500,
+    width: "",
+    bubbleColor: "",
+  };
   const historyComponent = shouldDisplayPortrait() ? (
-    <HistoryChat height={500} />
+    <Chat height={500} chatProps={videoChatProps} />
```

Comment:
you can probably just inline the `videoChatProps` into the `<Chat` comp and not have the const?